### DISCUSSION TITLE
Auto pre-heat step

### DISCUSSION THREAD

**Discussion by awwithro** [cooklang/spec#141] [General]
I'd be handy to have one baking step `bake for ~{30%minutes@350F}` and have that automatically result in the first step of the recipe being `1. Preheat oven to 350F` Probably the sort of thing that is much more tricky than it sounds :)

**Comment by TheMagicNacho** [comment]
How would we know that a preheat is required? Is it only for "bake for" keywords? Would there be other instances where pre-heat would be needed?
